lib/internal/source_map/source_map_cache.js

```diff
@@ -38,18 +37,22 @@ function maybeCacheSourceMap(filename, content, cjsModuleInstance) {
 
   const match = content.match(/\/[*/]#\s+sourceMappingURL=(?<sourceMappingURL>[^\s]+)/);
   if (match) {
+    const data = dataFromUrl(basePath, match.groups.sourceMappingURL);
+    const url = data ? null : match.groups.sourceMappingURL;
```

Comment:
It's getting to the point where we write quite a bit of data to disk -- we don't actually need the `url` field, if we've successfully parsed the map, so let's avoid storing it.